src/test/java/com/twilio/http/TwilioRestClientTest.java

```diff
@@ -2,19 +2,27 @@
 
 import com.twilio.Twilio;
 import com.twilio.rest.Domains;
-import mockit.Mocked;
+import mockit.Injectable;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.util.Arrays;
+
 import static org.junit.Assert.assertNotNull;
 
 public class TwilioRestClientTest {
-    @Mocked
+    @Injectable
     private TwilioRestClient twilioRestClient;
 
+    @Injectable
```

Comment:
Don't think this annotation is needed since `setUp` sets the value.

Reply:
Not able make method call without making variable as @Injectable
This was strange to me.